examples/TargetingConsoleApp/Program.cs

```diff
@@ -0,0 +1,75 @@
+﻿// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+//
+using Consoto.Banking.AccountService.Identity;
+using Microsoft.Extensions.Configuration;
+using Microsoft.Extensions.DependencyInjection;
+using Microsoft.FeatureManagement;
+using Microsoft.FeatureManagement.FeatureFilters;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+
+namespace Consoto.Banking.HelpDesk
+{
+    class Program
+    {
+        public static async Task Main(string[] args)
+        {
+            //
+            // Setup configuration
+            IConfiguration configuration = new ConfigurationBuilder()
+                .AddJsonFile("appsettings.json")
+                .Build();
+
+            //
+            // Setup application services + feature management
+            IServiceCollection services = new ServiceCollection();
+
+            services.AddSingleton(configuration)
+                    .AddFeatureManagement()
+                    .AddFeatureFilter<ContextualTargetingFilter>();
+
+            IUserRepository userRepository = new InMemoryUserRepository();
```

Comment:
>InMemoryUserRepository [](start = 49, length = 22)

I am thinking the idea to pass the user/group from command line or Thread.CurrentPrincipal?

Reply:
The purpose of the example is to show how to use the feature manager without relying on an ambient context.

Reply:
You could imagine that user references are being read in through a file and then full information on user is being pulled from database which in this case is abstracted through IUserRepository.